lang/cs/Org.Apache.REEF.Driver/Bridge/Events/AllocatedEvaluator.cs

```diff
@@ -109,7 +108,7 @@ public void SubmitContextAndServiceAndTask(IConfiguration contextConfiguration,
         {
             LOGGER.Log(Level.Info, "AllocatedEvaluator::SubmitContextAndServiceAndTask");
 
-            contextConfiguration = MergeContextConfiguration(contextConfiguration);
+            taskConfiguration = MergeWithConfigurationProviders(taskConfiguration);
```

Comment:
This is a workaround for [REEF-289](https://issues.apache.org/jira/browse/REEF-289), right? If so, please add a comment with that issue ID such that we can find it again in the future.